examples/webgl-context-lost/__tests__/index.js

```diff
@@ -0,0 +1,11 @@
+const { join } = require('path');
+
+it('should match the image snapshot', async () => {
+    const image = await page.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+it('should look good on the website!', async () => {
+    await page.goto('file://' + join(__dirname, '..', 'index.html'));
+    await page.screenshot({ path: join(__dirname, '..', 'screenshot.png') });
+});
```

Comment:
Is it possible to add automated testing of the various scenarios you've been testing with? Feel free to spin this out into an issue

Reply:
But.. please can you list the scenarios we should be supporting in the issue

Reply:
I've opened an issue: #1488